src/sriov.c

```diff
@@ -90,6 +127,13 @@ netplan_state_finish_sriov_write(const NetplanState* np_state, const char* rootd
                     pf = def;
                 else if (def->sriov_link)
                     pf = def->sriov_link;
+
+                if (pf) {
+                    if (pf->set_name)
+                        g_hash_table_add(apply_pfs, pf->set_name);
+                    else if (!pf->has_match) /* netdef_id == interface name */
+                        g_hash_table_add(apply_pfs, pf->id);
```

Comment:
nitpick: We might want to log some warning here in the "else" case where we don't have `set-name`, but still have a `match` stanza (i.e. when matching multiple interfaces). In which case it won't work.